containers/EditFly.js

```diff
@@ -14,7 +14,7 @@ class EditFly extends Component {
     }
 
     componentDidMount = () => {
-        const selectedFly = this.props.currentFlies.find(fly => fly.id === event.target.id)
```

Comment:
Great job fixing this! It has been an interesting journey going from React with the DOM to React Native and learning what items are no longer available to us since they are tied to the browser/DOM.